src/model.py

```diff
@@ -306,8 +313,10 @@ def prettyprint(self) -> str:
             result.append("Mutex " + mutex)
         for drop_filename in self.dropped_filenames:
             result.append("Drop " + drop_filename)
-        for email in self.emails:
-            result.append("Email " + email)
+        for email in self.emails_to:
+            result.append("Email To " + email)
+        for email in self.emails_from:
+            result.append("Email From " + email)
```

Comment:
```suggestion
            result.append("EmailFrom " + email)
```

For consistency and simpler parsing